tests/integration/helpers.py

```diff
@@ -65,6 +65,29 @@ def _install_proxy(conn: SSHConnection, proxy: ProxyConfig | None = None):
     assert result.ok, "Failed to configure iptable rules"
 
 
+def _install_dockerhub_mirror(conn: SSHConnection, dockerhub_mirror: str | None):
+    """Use dockerhub mirror if provided.
+
+    Args:
+        conn: The SSH connection instance.
+        dockerhub_mirror: The DockerHub mirror URL.
+    """
+    if not dockerhub_mirror:
+        return
+    command = f'echo {{ "registry-mirrors": ["{dockerhub_mirror}"]}} > /etc/docker/daemon.json'
```

Comment:
Should we also test the content of the `daemon.json` file? Something like this (I'm not sure if the syntax is correct)

```python
command = f'cat /etc/docker/daemon.json'
result: Result = conn.run(command)
assert dockerhub_mirror in result.stdout
```

Reply:
Since this is a helper command, I think it is okay to trust that the command works with `assert result.ok` check :D